File: 프로그래머스/1/42748. K번째수/K번째수.py
def solution(array, commands):
    answer = []
    
    for row in range(len(commands)) :
        i = commands[row][0]
        j = commands[row][1]
        k = commands[row][2]
        slice = array[i-1:j]
        slice.sort()
        answer.append(slice[k-1])
            
    return answer

File: 프로그래머스/1/42748. K번째수/K번째수.java
import java.util.*;

class Solution {
    public int[] solution(int[] array, int[][] commands) {
        
        int len = commands.length;
        int[] answer = new int [len];
        
        for(int i=0; i<len; i++){
            int start = commands[i][0];
            int end = commands[i][1];
            int num = commands[i][2];
            
            int [] arr = Arrays.copyOfRange(array, start-1, end);
            Arrays.sort(arr);
            answer[i] = arr[num-1];
        }

        return answer;
    }
}